database/migrations/2020_11_04_174749_create_book_copy_table.php

```diff
@@ -13,7 +13,7 @@ class CreateBookCopiesTable extends Migration
      */
     public function up()
     {
-        Schema::create('book_copies', function (Blueprint $table) {
+        Schema::create('book_copy', function (Blueprint $table) {
```

Comment:
Why the table name would be `book_copy`? It should be `book_copies` or `copies`.